pages/contribute.js

```diff
@@ -273,10 +273,15 @@ class TiersPage extends React.Component {
                       <H2 fontWeight="normal" mb={2}>
                         {title}
                       </H2>
-                      {LoggedInUser?.canEditCollective(collective) && verb === 'events' && (
-                        <Link href={`/${collective.slug}/events/new`}>
+                      {LoggedInUser?.canEditCollective(collective) && (verb === 'events' || verb === 'projects') && (
+                        <Link href={`/${collective.slug}/${verb}/new`}>
                           <StyledButton buttonStyle="primary">
-                            <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                            {verb === 'events' && (
+                              <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                            )}
+                            {verb === 'projects' && (
+                              <FormattedMessage id="SectionProjects.CreateProject" defaultMessage="Create Project" />
+                            )}
```

Comment:
I think just duplicating the event's code would look simpler but this also works.

Reply:
Ah, I thought about this initially and then thought the other approach has less duplicated lines. Anyways, I am okay with this as well and therefore changed the code to this; it definitely looks simpler. 👍🏽